18_links_lab/linkslab.txt

```diff
@@ -8,13 +8,15 @@ Preparation: Perform the following  (no need to write the commands you used):
 
   - Create a symbolic link named file3 that points to "file1"
   Answer: 
-
+cd links
+ln -s ./file1 ./file3
   - Create a hard link named file4 that points to "file2"
   Answer: 
-
+ln file2 file4
   - Create a new directory named "tmp" in your home folder:
   Answer:
 
 2. Move "file3" to the new "~/tmp" directory you just created. Can you still use it to see the original "file1" ? Why not? How would you create a symbolic link that would survive the transfer to ~/tmp ?
-
+no, we changed file3 place so the link will not work anymore. we can use "ls -l" which survive thee transfer. 
 3. How can you tell that "file2" and "file4" refer to the same file?
+ls -l [option to sort] and then we can the last date of Modifying and also how names there are to Specific file. 
```

Comment:
Better to use `ls -li` - as that prints also the special "inode index" of the file. A unique number that equals only for hard links